spec/features/editing_file_attachments/delete_file_attachment_spec.rb

```diff
@@ -15,23 +25,45 @@ def given_there_is_an_edition_with_attachments
                       file_attachment_revisions: [@attachment_revision])
   end
 
+  def given_there_is_an_edition_with_featured_attachments
+    @attachment_revision = create(:file_attachment_revision, :on_asset_manager)
```

Comment:
This shouldn't be necessary for this feature. I notice it's in the original scenario too - it should be safe to remove.

Reply:
Which bit?

Reply:
Ah, sorry: `on_asset_manager`. It doesn't change the outcome from the feature's perspective - it doesn't have any Asset Manager stubs.